router/broker.go

```diff
@@ -555,3 +554,12 @@ func (b *Broker) pubSubCreateMeta(subTopic wamp.URI, subSessID, subID wamp.ID, m
 	}
 	b.pubMeta(wamp.MetaEventSubOnCreate, sendMeta)
 }
+
+func (b *Broker) trySend(sess *wamp.Session, msg wamp.Message) error {
```

Comment:
instead of having this function in both, the `broker` and the `dealer` just for logging purpose, have you considered allowing the various `broker` and `dealer` methods return the error and handle the logging in the `realm`?

The error from `trySend` is just being thrown away anyway.

Reply:
Problem with returning the message is that it would need to be returned via a channel to the realm (or session in realm).  Broker or dealer cannot call realm either since they do not know about the parent realm.   Still, yes, there probably is a better way that having the same code in two places.